docker/daemon_unix.go

```diff
@@ -11,7 +11,7 @@ import (
 	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/pkg/system"
 
-	_ "github.com/docker/docker/daemon/execdriver/lxc"
+	//"github.com/docker/docker/daemon/execdriver/lxc"
```

Comment:
It pulls in the driver code -- without it, lines like https://github.com/docker/docker/blob/06b922c63f660be431e4b678a50ae1ddb0dcf882/daemon/execdriver/lxc/init.go#L38 don't ever get executed, so it either needs to stay in here, or be moved to a Linux-specfic file in this same package.